web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1665,6 +1665,11 @@
       $scope.isUserGroupUpdateEnabled = gnGlobalSettings.isUserGroupUpdateEnabled;
       $scope.isExternalViewerEnabled = gnExternalViewer.isEnabled();
       $scope.externalViewerUrl = gnExternalViewer.getBaseUrl();
+      $scope.publicationOptions = [];
+
+      $http.get("../api/records/publicationOptions").then(function (response) {
```

Comment:
Should we move the endpoint to something like `/sharing/modes` or `sharing/options` as it relates more to general sharing configuration than to records ?

![image](https://github.com/geonetwork/core-geonetwork/assets/1701393/5e7fbdab-c268-4db3-8440-7f656e6e24ba)

Reply:
Updated